app/views/facility_accounts/show_statement.html.haml

```diff
@@ -38,7 +38,7 @@
                 %li.indented= link_to 'View Order Form', od.external_service_receiver.show_url, :target => '_blank'
               - if !od.stored_files.template_result.empty?
                 - od.stored_files.template_result.each do |stored_file|
-                  %li.indented= link_to 'View Order File', stored_file.file.url
+                  %li.indented= link_to 'View Order File', stored_file.download_url
```

Comment:
I think it is obsolete, but we should probably leave it in anyways. Although, it looks like you triggered a couple exceptions when trying to view this on stage. We might want to fix that up at some point.